net/nimble/host/src/ble_gatts.c

```diff
@@ -2080,6 +2080,141 @@ ble_gatts_count_cfg(const struct ble_gatt_svc_def *defs)
     return 0;
 }
 
+static const char* ble_gatt_chr_f_names[] = {
```

Comment:
The pointed-to strings are const, but the pointers themselves aren't.  The pointers should probably be made const so that this array can be placed in .text rather than .data.

E.g.,

`static const char * const ble_gatt_chr_f_names[]`

The same comment applies to the ble_gatt_dsc_f_names array below.